detectron2/data/datasets/coco.py

```diff
@@ -331,7 +331,8 @@ def convert_to_coco_dict(dataset_name):
                 area = polygons.area()[0].item()
             else:
                 # Computing areas using bounding boxes
-                area = Boxes([bbox]).area()[0].item()
+                bbox_xy = BoxMode.convert(bbox, bbox_mode, BoxMode.XYXY_ABS)
```

Comment:
this is incorrect. Since `bbox` already becomes `XYWH_ABS` mode due to line 323, the "from mode" in this line should be "XYWH_ABS", instead of `bbox_mode`.